src/rosconsole/impl/rosconsole_log4cxx.cpp

```diff
@@ -379,7 +379,7 @@ void shutdown()
     g_log4cxx_appender = 0;
   }
   // reset this so that the logger doesn't get crashily destroyed
-  // again during global destruction.  
+  // again during global destruction.
```

Comment:
Please revert unrelated whitespace changes like this.

Reply:
reverted